src/tests/createHeadlessForm.test.js

```diff
@@ -2506,17 +2503,15 @@ describe('createHeadlessForm', () => {
   });
 
   describe('when a field has conditional presentation properties', () => {
-    it('adds .jsf-statement to nested statement markup when visible', () => {
+    it('adds the html to nested statement markup when visible', () => {
```

Comment:
todo: Saying this test "adds the html" is misleading because the HTML always came from the JSON Schema. The only thing it does is to pass over the property as it is.

```suggestion
    it('returns the nested properties when the conditional matches', () => {
```

Reply:
Changed!